### ISSUE TITLE
Overload pipeline of model hosted on triton server  

### ISSUE THREAD

**Issue by leopra** [ChainYo/transformers-pipeline-onnx#4]
I'm quite confused on how to implement  this. Once converted the ner model to onnx I want to deploy it to a triton server. The issue is that I would like to get the full inference on the triton server but it looks like I can just receive the logits and then locally compute the entities. 
Is there a way to "send" the Overcharged TokenClassificationPipeline to the triton server so that the Triton Inference call directly returns the dictionary of entities?
